### ISSUE TITLE
Dashed separator transparency

### ISSUE THREAD

**Issue by dunst** [DaveDavenport/rofi#262]
Hello, I tried to set solid separator line, but my separator is dashed anyway . And dashed line has 100% transparency. It's very ugly.

``` bash
rofi.opacity: 95
rofi.lines: 7
rofi.columns: 1
rofi.font: Ubuntu Mono Bold 12
rofi.fg: #E5DBCF
rofi.bg: #0F3B5F
rofi.hlfg: #0F3B5F
rofi.hlbg: #CC9752
rofi.bc: #CC9752
rofi.bw: 2
rofi.separator-style: solid
rofi.fixed-num-lines: true
rofi.hmode: false
rofi.disable-history: true
```

![Screenshot](http://i.imgur.com/Bs7cXYs.png)

**Comment by DaveDavenport** [comment]
That is a curious bug. Solid line is working here, also the transparency on dashed.

What version are you running?
Can you pastebin `rofi -dump-xresources`

**Comment by dunst** [comment]
http://pastebin.com/0Wiw6CNP

**Comment by DaveDavenport** [comment]
What version are you running?  there was a big overhaul in drawing in 0.15.11, so I suspect this might be fixed now.

**Comment by dunst** [comment]
$ rofi -v              
Version: 0.15.7

**Comment by DaveDavenport** [comment]
In 0.15.10 (sorry said 0.15.11 before) rofi moved to cairo for drawing, this should have resolved this problem.
Is it possible to verify this with git? (0.15.10 has some bugs,)

**Comment by dunst** [comment]
Thanks, It was fixed in 0.15.10
